x/pocketcore/types/cache.go

```diff
@@ -238,7 +245,8 @@ func (ei *EvidenceIt) Value() (evidence Evidence) {
 	// unmarshal the value (bz) into an evidence object
 	err := ModuleCdc.UnmarshalJSON(ei.Iterator.Value(), &evidence)
 	if err != nil {
-		panic(fmt.Errorf("can't unmarshal evidence iterator value into evidence: %s", err.Error()))
+		fmt.Println(fmt.Errorf("can't unmarshal evidence iterator value into evidence: %s", err.Error()))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("can't unmarshal evidence iterator value into evidence: %s", err.Error())
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)